wsr-tic-tac-toe/src/redux/reducers/index.ts

```diff
@@ -0,0 +1,137 @@
+import {
+    WinningScenarioActionTypes,
+    BoardState,
+    BoardStateActionTypes,
+    SquareState,
+    GAME_STATE,
+} from '../types';
+import {
+    CLEAR_WINNING_SCENARIOS,
+    CREATE_BOARD,
+    REMOVE_BOARD,
+    GENERATE_WINNING_SCENARIOS,
+    SELECT_SQUARE,
+    CLEAR_BOARD,
+    SET_GAME_STATE,
+} from '../actions';
+import { combineReducers } from 'redux';
+const initialWinningScenariosState: Array<Array<number>> = [];
+
+export function winningScenariosReducer(
```

Comment:
it's generally good practise to put reducers in individual files, and then have an index file that combines them